src/schema.ts

```diff
@@ -261,31 +272,65 @@ UserSchema.virtual("oldEnoughForWithdrawal").get(function (this: typeof UserSche
   return d - this.created_at.getTime() > yamlConfig.limits.oldEnoughForWithdrawal
 })
 
-UserSchema.methods.limitHit = async function ({
-  on_us,
-  amount,
-}: {
-  on_us: boolean
-  amount: number
-}) {
-  const timestampYesterday = Date.now() - MS_PER_DAY
+UserSchema.virtual("twoFactorEnabled").get(function (this: typeof UserSchema) {
+  return !!this.twoFactor.secret
```

Comment:
Not a fan of `!!`. If we don't have a helper for this logic, maybe introduce something like `hasValue(this.twoFactor.secret)`

Reply:
Checking for `x != null` seems to be a common way in JS/TS, so I've refactored this check that way. Wdyt?